CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing
+
+Thank you for your interest in contributing to this project! Here are some guidelines to get you started.
+
+## Pre-requisites
+
+The easiest way to get started is to use [Docker](https://www.docker.com/) with [Docker Compose](https://docs.docker.com/compose/).
+This project has a `compose.yaml` file which will set up a development environment for you.
+
+## Setup
+
+1. Clone this repository
+2. Clone the [Basic_Framework](https://github.com/SjonHortensius/Basic_Framework) repository
+  - TODO: Simplify this step
+3. Copy `config.ini.template` to `config.ini`
```

Comment:
I suggest this step, so the `config.ini` file can be `.gitignore`d, what do you think?

Reply:
I agree but prefer `config.example.ini` as it matches the framework